minutus-mruby-build-utils/src/lib.rs

```diff
@@ -86,16 +95,42 @@ impl MRubyManager {
 }
 
 fn build_mruby(workdir: &Path, path: &Path) {
+    let rake = match () {
+        #[cfg(windows)]
+        () => "rake.bat",
+        #[cfg(not(windows))]
+        _ => "rake",
+    };
+
```

Comment:
This is a bit comprecated.
Could we write this as follows?

```suggestion
    let rake = if (cfg!(windows)) {
      "rake.bat"
    } else {
      "rake"
    };
```

This change might add runtime overhead, but I think it's negligible.

Reply:
I updated it to `if cfg!(windows)` since the outer parentheses in `if (cfg!(windows))` are unnecessary.